.gitignore

```diff
@@ -1,4 +1,6 @@
 .idea/
+!.idea/copyright/BSD_3_Clause.xml
```

Comment:
Is there a trick to using these files correctly? When I created a new project in IntelliJ for this repo, it just blew them away.